vpr/src/draw/draw.cpp

```diff
@@ -4157,4 +4135,16 @@ static void run_graphics_commands(std::string commands) {
     ++draw_state->sequence_number;
 }
 
+#ifdef VTR_ENABLE_DEBUG_LOGGING
+// location highlighting functions 
```

Comment:
Expand this:  pass in an (x,y,subtile) location and the color in which it should be drawn.  This overrides the color of any block placed in that location, and also applies if the location is empty.